docs/virtual_machines/accessing_virtual_machines.md

```diff
@@ -290,21 +216,170 @@ spec:
           propagationMethod:
             qemuGuestAgent:
               users:
-              - "fedora"
+              - fedora
       volumes:
-      - dataVolume:
-          name: fedora-dv
-        name: disk0
+      - containerDisk:
+          image: quay.io/containerdisks/fedora:latest
+        name: containerdisk
       - cloudInitConfigDrive:
-          userData: |
-            #!/bin/bash
+          userData: |-
+            #cloud-config
+            password: fedora
+            chpasswd: { expire: False }
+            # Disable SELinux for now, so qemu-guest-agent can write the authorized_keys file
+            bootcmd:
+              - setenforce 0
```

Comment:
Can we avoid using the SELinux permissive mode by assigning the appropriate SELinux context?
Something similar to:
 `- ["sudo", "chcon", "--type", "virt_qemu_ga_exec_t", "/home/$USER/.ssh/authorized_keys"]`

Reply:
The issue lies deeper, out of the context for this example.

`audit2why` says:

```
type=AVC msg=audit(1665750224.275:505): avc:  denied  { dac_read_search } for  pid=674 comm="qemu-ga" capability=2  scontext=system_u:system_r:virt_qemu_ga_t:s0 tcontext=system_u:system_r:virt_qemu_ga_t:s0 tclass=capability permissive=0
type=AVC msg=audit(1665750224.275:506): avc:  denied  { dac_override } for  pid=674 comm="qemu-ga" capability=1  scontext=system_u:system_r:virt_qemu_ga_t:s0 tcontext=system_u:system_r:virt_qemu_ga_t:s0 tclass=capability permissive=0
```

I would rather keep disabling SELinux for now. (although I really don't like to)

Reply:
I agree.